contribute/docs/index.src.html

```diff
@@ -67,19 +84,3 @@ <h2>/*pad()*/</h2>
 <pre><code>echo 12345;          // 12345
 echo 6;              // 6
 </pre></code></p>
-
-<p>Apart from the docs the typical steps can be used to contribute your own improvements.</p>
-
-<ul>
-    <li>Fork</li>
-    <li>Clone</li>
-    <li>PHPUnit</li>
-    <li>Branch</li>
-    <li>PHPUnit</li>
-    <li>Code</li>
-    <li>PHPUnit</li>
-    <li>Commit</li>
-    <li>Push</li>
-    <li>Pull request</li>
-    <li>Relax and play Castle Crashers</li>
-</ul>
```

Comment:
Woh, that's very old, obsolete and PHPUnit is not even installed in the gh-pages branch.

Reply:
I fixed that on your branch directly.